docs/DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1 @@
+todo
```

Comment:
Can add a few quick comments on how to setup minikube to run this controller? 

Is this accurate?..

```suggestion
## Build controller
make docker-build

## Setup on minikube
1. Install knative: https://www.knative.dev/docs/install/knative-with-minikube/
2. Deploy controller manager: make deploy
```

Reply:
Cut a PR :)